### PR TITLE
Integrate po2vlan testcases into general qualification pipeline

### PR DESCRIPTION
**Pull request by ganglyu**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
Integrate po2vlan testcases into general qualification pipeline

#### How did you do it?
Update po2vlan related test cases, create new portchannel and add it to two VLANs.

#### How did you verify/test it?
Run end to end test on t0 testbed.

### CODE REVIEW

File: tests/common/helpers/portchannel_to_vlan.py [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+import sys
+from netaddr import valid_ipv4
+
+from tests.common.helpers.assertions import pytest_require
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses    # noqa F401
+from tests.common.fixtures.duthost_utils import ports_list   # noqa F401
+from tests.common.fixtures.duthost_utils import utils_vlan_intfs_dict_orig          # noqa F401
+from tests.common.fixtures.duthost_utils import utils_vlan_intfs_dict_add
+from tests.common.helpers.backend_acl import apply_acl_rules, bind_acl_table
+from tests.generic_config_updater.gu_utils import create_checkpoint, rollback
```
Comment (wangxin) [inline tests/common/helpers/portchannel_to_vlan.py:15]: Can you also move the shared functions from `tests.generic_config_updater` to `tests/common'?

Comment (ganglyu) [reply tests/common/helpers/portchannel_to_vlan.py]: Updated.

**Review by wangxin — APPROVED** [review]

**Comment by bingwang-ms** [comment]
@ganglyu  Do we need this change in `202205` and `202012` branches?

**Comment by ganglyu** [comment]
> @ganglyu Do we need this change in `202205` and `202012` branches?

I have merged this fix in 202205, and we don't need it in 202012.

**Comment by bingwang-ms** [comment]
> > @ganglyu Do we need this change in `202205` and `202012` branches?
> 
> I have merged this fix in 202205, and we don't need it in 202012.

Can you please share the PR link for 202205?

**Pull request merged**